runtime/dart_isolate.cc

```diff
@@ -348,9 +348,18 @@ bool DartIsolate::PrepareForRunningFromKernel(
     return false;
   }
 
-  child_isolate_preparer_ = [mapping](DartIsolate* isolate) {
-    return isolate->PrepareForRunningFromKernel(mapping);
-  };
+  if (child_isolate_preparer_  == nullptr) {
```

Comment:
Could you add a comment here explaining how we might get here more than once, and why we don't want to reinitialize child_isolate_preparer_ in that case?

Reply:
Done